src/app/containers/CpsAssetPageMain/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { string, shape, object, arrayOf } from 'prop-types';
+import { Grid, GridItemConstrainedMedium } from '#lib/styledGrid';
+
+const CpsAssetPageMain = () => {
+  return (
+    <Grid as="main" role="main">
+      <GridItemConstrainedMedium>
+        <h1> Placeholder content for MAP page skeleton</h1>
+      </GridItemConstrainedMedium>
+    </Grid>
+  );
+};
+
+CpsAssetPageMain.propTypes = {
+  /* eslint-disable react/no-unused-prop-types */
+  pageData: shape({
```

Comment:
question, is there no way we could reuse most of the proptypes in `#models/`? like [here](https://github.com/bbc/simorgh/blob/latest/src/app/models/propTypes/radioPage/index.js#L6)